### ISSUE TITLE
adjust frequency bands and add more?

### ISSUE THREAD

**Issue by Stephanowicz** [raedwulf/alsaequal#8]
Hi,

I wanted to know if there's a possibility to adjust the frequencies of the 10 bands?
(btw. how are the frequencies treated, that are between the band controllers?)
And of what I've seen in the ladspa_utils.c there is a maximum of 16 channels - so is it possible to display and adjust more than 10 channels?

Thx & cheers,

Stephan

**Comment by bennigraf** [comment]
Hey,
`alsaequal` uses the ["Caps" LADSPA plugin suite](http://quitte.de/dsp/caps.html#Eq10) and their Eq10 plugin, which defines the frequencies of the 10 bands. You cannot change those without changing the code of the caps plugins. You could alternatively use another plugin, though, i.e. Eq4p of CAPS or something from a completely different library, by using the "library" and "module" configuration as described in the Readme. I was doing something similar with success.
I hope this helps!

**Comment by Stephanowicz** [comment]
Thankyou @bennigraf,
may I ask what kind of lib You did use?

**Comment by bennigraf** [comment]
Hey! I used the Caps library linked above, which is also the default for this plugin (as stated in the readme).

**Comment by Stephanowicz** [comment]
Ok - maybe I misunderstood... "I was doing something similar with success." sounded to me like You did use another lib than the Caps lib...? :D

**Comment by bennigraf** [comment]
No, it really was just Eq4p from Caps. But I don't see why it wouldn't work with others as well.

**Comment by Stephanowicz** [comment]
Ok, ok :) 
maybe sometime I will dig deeper into the caps lib.
Would still be interesting to know how the frequencies between the bands are treated and if one could adjust this - e.g the 'width'  and intensity of change around the adjusted frequency band

**Comment by bennigraf** [comment]
In Eq10 of the Caps library the frequency bands are kind of hardcoded apparently, but I found the "TAP-plugins" (https://github.com/tomszilagyi/tap-plugins) – they contain the "TAP Equalizer", which has 10 bands with adjustable frequencies. There's some documentation here: https://tomscii.sig7.se/tap-plugins/ladspa/eq.html – if you get this running with alsaequal, let me know, I'd be interested as well!

**Comment by Stephanowicz** [comment]
Hi Benni, thank You!
Yes, I will have a look - but @moment I'm busy with other things... dunno when I will have time for this

Cheers, Stephan
